src/graphql.client/AlkemioCliClient.ts

```diff
@@ -70,10 +70,16 @@ export class AlkemioCliClient {
 
   public async ingestSpace(spaceID: string) {
     const result = await this.sdkClient.spaceIngest({ spaceID });
-
     return result.data.lookup.space;
   }
 
+  public async ingestKnowledgeBase(knowledgeBaseID: string) {
+    const result = await this.sdkClient.knowledgeBaseIngest({
+      knowledgeBaseID,
+    });
+    return result.data.lookup.knowledgeBase;
```

Comment:
I am interested in how the client handles graphql errors since I don't see an `error` key in the response definition.

Reply:
I think the sdkClient will ultimately throw if something is wrong with the query.